src/libs/EmojiUtils.js

```diff
@@ -77,7 +78,7 @@ function isSingleEmoji(message) {
  * @returns {Boolean}
  */
 function containsOnlyEmojis(message) {
-    const trimmedMessage = message.replace(/ /g, '').replaceAll('\n', '');
+    const trimmedMessage = Str.replaceAll(message.replace(/ /g, ''), '\n', '');
```

Comment:
it's poly-filled and not available on some mobile browsers

Reply:
Cool ya makes sense

Any thoughts on my second comment?

> Also, `.replace(/ /g, '')` is basically another `.replaceAll(message, ' ', '')` so why not replace that one too?

Reply:
Mostly added this to fix a bad crash with iOS 14 so that wasn't my priority. We could change the usages, but that doesn't stop people from making this mistake in the future. I also added an eslint rule to prevent people from using `string.prototype.replaceAll`.